### PR TITLE
Swap & d-s-s & Docker-latest support

### PR DESCRIPTION
**Pull request by cevich**
This will probably fail CI (it depends on changes there also)

### CODE REVIEW

**Comment by cevich** [comment]
[(?) [CI] [CN] Add integration test to verify selinux labels](https://trello.com/c/oAZFBed5/310-ci-cn-add-integration-test-to-verify-selinux-labels)

File: roles/sysconfig_docker/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    replace: "DOCKERBINARY=/usr/bin/docker-latest"
+  when: sysconfig_docker_latest is defined and
+        sysconfig_docker_latest
+
+- name: DOCKERBINARY commented out
+  replace:
+    backup: True
+    dest: /etc/sysconfig/docker
+    regexp: "^(#\\s*)?(DOCKERBINARY=.*)"
+    replace: "# \\1"
```
Comment (edsantiago) [inline roles/sysconfig_docker/tasks/main.yml]: Should this be \\2 ?

Comment (cevich) [inline roles/sysconfig_docker/tasks/main.yml]: Oh excellent, you're right!  Thanks!

**Comment by cevich** [comment]
ho-hum...dang networking timeouts.

**Comment by cevich** [comment]
Uggg, I dunna get it.  It's waiting for port 22 to come up on every peon, but then failing to run setup on one/more.  Though, you get the same error if there's a problem with ssh-keys or username, hmmmm...

**Comment by cevich** [comment]
jenkins, test this please.

**Comment by cevich** [comment]
jenkins, test this please.

**Comment by cevich** [comment]
jenkins, test this please.

**Pull request merged**